docs/source/quickstart.R

```diff
@@ -3,5 +3,5 @@ library(opendp)
 enable_features("contrib")
 # 2-use
 space <- c(atom_domain(.T = "f64"), absolute_distance(.T = "f64"))
-base_laplace <- space |> then_base_laplace(1.)
-dp_agg <- base_laplace(arg = 23.4)
\ No newline at end of file
+m_laplace <- space |> then_laplace(1.)
+dp_agg <- m_laplace(arg = 23.4)
```

Comment:
Particularly on the examples, I'd really prefer to to spell out the variable names. This is their first introduction to OpenDP.